lib/web_ui/lib/src/engine/compositor/color_filter.dart

```diff
@@ -0,0 +1,31 @@
+part of engine;
+
+class SkColorFilter {
+  js.JsObject skColorFilter;
+
+  SkColorFilter.mode(EngineColorFilter filter) {
+    skColorFilter =
+        canvasKit['SkColorFilter'].callMethod('MakeBlend', <dynamic>[
+      filter._color.value,
+      makeSkBlendMode(filter._blendMode),
+    ]);
+  }
+
+  SkColorFilter.matrix(EngineColorFilter filter) {
+    final js.JsArray colorMatrix = js.JsArray();
+    colorMatrix.length = 20;
+    for (int i = 0; i < 20; i++) {
+      colorMatrix[i] = filter._matrix[i];
```

Comment:
Let's leave a todo to remove this double-conversion. This should, at least, convert straight to a `Float64List`, but ideally we should write to WASM memory directly.

/cc @kjlubick - here's an example for excessive conversion that we talked about at the last sync.

Reply:
Done.